app/models/relationship.rb

```diff
@@ -0,0 +1,5 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: User.name
+  belongs_to :followed, class_name: User.name
+  scope :follower_of_user, ->user_id{where follower_id: user_id}
```

Comment:
Bad name
I think `followers` is a suitable name for this `scope`

Reply:
Yes